SwiftI18n/Classes/Views/BaseViews/UIBarButtonItem+I18n_loc.swift

```diff
@@ -10,16 +10,40 @@ import UIKit
 public extension UIBarButtonItem {
     
     static let loc_titleKey = "KEY"
-    
+    static let loc_accessibilityLabelKey = "ACCESSIBILITY_LABEL_KEY"
+    static let loc_accessibilityHintKey = "ACCESSIBILITY_HINT_KEY"
+
     @IBInspectable var locTitleKey: String? {
         get {
-            return loc_keysDictionary[UIViewController.loc_titleKey]
+            return loc_keysDictionary[UIBarButtonItem.loc_titleKey]
         }
         
         set(newValue) {
-            loc_keysDictionary[UIViewController.loc_titleKey] = newValue
+            loc_keysDictionary[UIBarButtonItem.loc_titleKey] = newValue
```

Comment:
Before we go with this change, @Truba, do you remember why we used `UIViewController.loc_titleKey` here instead of `UIBarButtonItem.loc_titleKey`?

From what I can see, we are doing that on other places as well (e.g., UINavigationItem), so let's either return this change to its previous state or see if we can update other places as well.